app/celery/scheduled_tasks.py

```diff
@@ -220,3 +221,46 @@ def check_templated_letter_state():
                 message=msg,
                 ticket_type=zendesk_client.TYPE_INCIDENT,
             )
+
+
+@notify_celery.task(name="heartbeart-inbox-sms")
+@statsd(namespace="tasks")
+def heartbeat_inbox_sms():
+    """
+    The function acts as a heartbeat to the inbox queue. Inbox is a list in Redis that
+    is made up of notifications. The post_api will push all the notifications to Inbox.
```

Comment:
Let's avoid mentioning the inbox and refer to the queue simply, as this is an internal implementation detail to the `RedisQueue` and that could change in the future.